install.js

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/fluid-project/first-discovery-server/raw/master/LICENSE.txt
+*/
+
+"use strict";
+
+var child_process = require("child_process");
+
+// spawns the npm install
+// need to use --force to ensure that "dedupe-infusion" failures from gpii-express
+// do not prevent its installation.
+var npm = child_process.spawn("npm", ["install", "--force"], {stdio: "inherit"});
```

Comment:
Looks like this repeated use of spawn already merits being factored out into a function.
It's worth adding to the comment for this function that we use spawn rather than exec because we are uninterested in the buffer from exec and it may overflow - useful guide here: http://www.hacksparrow.com/difference-between-spawn-and-exec-of-node-js-child_process.html